doc/man7/provider-decoder.pod

```diff
@@ -41,13 +41,14 @@ provider-decoder - The OSSL_DECODER library E<lt>-E<gt> provider functions
 
 =head1 DESCRIPTION
 
-I<We use the wide term "decode" in this manual.  This includes but is
-not limited to deserialization.>
+I<The term "decode" is used throughout this manual.  This includes but is
+not limited to deserialization as individual decoders can do also
```

Comment:
also do

Reply:
Fixup pushed. I assume no re-approval is needed.

Reply:
I must be tired - it took me a few gos to put the emoji in the correct place..